packages/vscode-extension/src/webview/components/Preview.tsx

```diff
@@ -487,7 +487,7 @@ function Preview({
   }, [project, openRebuildAlert, projectStatus]);
 
   const device = iOSSupportedDevices.concat(AndroidSupportedDevices).find((sd) => {
-    return sd.name === projectState?.selectedDevice?.name;
+    return sd.displayName === projectState?.selectedDevice?.name;
```

Comment:
we want to accept only supported devices detecting them by device names, not the display names, which users will be able to change after #597